apps/dashboard/src/app/(dashboard)/(chain)/[chain_id]/[contractAddress]/account/page.tsx

```diff
@@ -0,0 +1,26 @@
+import { notFound } from "next/navigation";
+import { AccountPage } from "../../../../../../contract-ui/tabs/account/page";
+import { getContractPageParamsInfo } from "../_utils/getContractFromParams";
+import { getContractPageMetadata } from "../_utils/getContractPageMetadata";
+
+export default async function Page(props: {
+  params: {
+    contractAddress: string;
+    chain_id: string;
+  };
+}) {
+  const info = await getContractPageParamsInfo(props.params);
+
+  if (!info) {
+    notFound();
+  }
+
+  const { contract, chainMetadata } = info;
+  const { isAccount } = await getContractPageMetadata(contract);
+
+  if (!isAccount) {
+    notFound();
```

Comment:
means its a contract but not a contract that supports this, I think the pattern we want in that case is to just redirect to the root of the contract page

Reply:
Yep, makes sense. Changing

Reply:
changed